src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt

```diff
@@ -52,7 +49,7 @@ open class ModernMethodCountTask: DexMethodCountTaskBase() {
 
 @ParallelizableTask
 open class LegacyMethodCountTask: DexMethodCountTaskBase() {
-
+    @OutputFile
```

Comment:
We used to have `@OutputFile`, `@Input`, and similar here, but I removed them; they opt in to Gradle's cache, which causes crashes due to broken Java serialization.  The task itself is serializable, but the most important bit of input (the BaseVariantOutput) is _not_.  Anyways, without marking `variantOutput` this task will be incorrectly up-to-date more often than not.  If we _do_ mark it as an input, we'll get crashes.

I think we should just not do this change.